pkg/publish/daemon.go

```diff
@@ -145,7 +145,8 @@ func (d *demon) Publish(ctx context.Context, br build.Result, s string) (name.Re
 	}
 
 	log.Printf("Loading %v", digestTag)
-	if _, err := daemon.Write(digestTag, img, d.getOpts(ctx)...); err != nil {
+	if resp, err := daemon.Write(digestTag, img, d.getOpts(ctx)...); err != nil {
+		log.Println("daemon.Write response: ", resp)
```

Comment:
Even if `err == nil`, there's sometimes interesting stuff in resp. Should we care? Wasn't your debugging purgatory one such situation?

Reply:
Yeah 😒 

The problem is that resp is pretty noisy in the common case where it actually works and doesn't return an err (it logs write progress bars and some other stuff), so I wouldn't want to log it all the time. Maybe only if `--verbose`, but then I'd have to add a `--verbose` flag.

I'm going to tempt fate and say that hopefully this fairly arcane situation is one of the only ones that cause resp to be useful in the absence of err, and try to remember that sometimes it's the case if I'm unlucky enough to find myself debugging arcane `daemon.Write` issues in the future.